www/docs/src/content/docs/start-here/getting-started.mdx

```diff
@@ -19,6 +16,12 @@ To start using Astro StudioCMS, you'll need:
 
 ## Creating an Astro Studio account
 
+:::note
+Currently, Astro Studio is in closed beta, and you'll need an invite to create an account. 
+
+While we understand this may be frustrating, we're following Astro Studio's News and Updates for the latest information on when the platform will be available to the public again.  There is also an open [PR for `@astrojs/db`](https://github.com/withastro/astro/pull/11385) that would allow you to use the Astro StudioCMS integration without the need for Astro Studio by using another LibSQL database service or even self-hosted.
```

Comment:
```suggestion
While we understand this may be frustrating, we're following Astro Studio's News and Updates for the latest information on when the platform will be available to the public again. There is also an open [PR for `@astrojs/db`](https://github.com/withastro/astro/pull/11385) that would allow you to use the Astro StudioCMS integration without the need for Astro Studio by using another LibSQL database service or even self-hosted.
```